bills.toml

```diff
@@ -4,3 +4,6 @@ megabytes = 20.00
 devices = 42.00
 extras = 1.00
 fees = 12.84
+deviceIds = [ "1112223333", "1112224444", "1112220000" ]
```

Comment:
Lucky enough, my account is currently in the state that inspired this.  My brother is out of the country, so his line isn't incurring any cost on these maps.  However, it still incurs the monthly device and extra fees that get split evenly across all lines.